blueprints/authenticator/index.js

```diff
@@ -15,25 +15,25 @@ module.exports = {
 
     if (baseClass === 'torii') {
       return {
-        imports: 'import Ember from \'ember\';' + EOL + 'import Torii from \'ember-simple-auth/authenticators/torii\';',
+        imports: 'import Torii from \'ember-simple-auth/authenticators/torii\';',
         baseClass: 'Torii',
         body: EOL + '  torii: Ember.inject.service(\'torii\')'
```

Comment:
I removed that line because I received a jslint error about the unused import. Not sure what the difference is between your setup and mine is, then.